src/macaron/config/defaults.py

```diff
@@ -37,22 +36,19 @@ def get_list(
         If ``strip`` is True  (default: True), strings are whitespace-stripped and empty strings
         are removed from the final result.
 
-        If ``duplicated_ok`` is True (default: False), duplicated values are not removed from the final list.
-
+        The order of non-empty elements in the list is preserved.
         The content of each string in the list is not validated and should be handled separately.
 
         Parameters
         ----------
         section : str
             The section in ``defaults.ini``.
-        item : str
-            The item to parse the list.
+        option : str
```

Comment:
Nit: I would call this `key` instead of `option`, as suggested by the [configparser documentation](https://docs.python.org/3/library/configparser.html#supported-ini-file-structure) and [wikipedia](https://en.wikipedia.org/wiki/INI_file).

Reply:
I think the reason I decided to use `option` is because I was following a similar interface as these `get` method in the `configparser` documentation - https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.get
But if it's more correct to use the same terminology for the INI file here then I am happy to change too.

Reply:
I see. I think if you have chosen it for a reason then I think it's fine.